README.md

```diff
@@ -1,21 +1,48 @@
+<!--
+:type: service
+:name: TICK Stack GCP
+:description: Deploy the TICK stack (Telegraf, InfluxDB, Chronograf, Kapacitor) in GCP to gather and process time series data.
+:icon: /_docs/tick-stack-gcp-icon.png
+:category: Other data stores
+:cloud: gcp
+:tags: database, time-series
+:license: open-source
+:built-with: terraform, bash
+-->
+# TICK Stack GCP Module
+
 [![Maintained by Gruntwork.io](https://img.shields.io/badge/maintained%20by-gruntwork.io-%235849a6.svg)](https://gruntwork.io/?ref=repo_google_influx)
 [![GitHub tag (latest SemVer)](https://img.shields.io/github/tag/gruntwork-io/terraform-google-influx.svg?label=latest)](https://github.com/gruntwork-io/terraform-google-influx/releases/latest)
 ![Terraform Version](https://img.shields.io/badge/tf-%3E%3D0.12.0-blue.svg)
-# TICK Stack GCP Module
 
 This repo contains the **official** module for deploying the [TICK Stack](https://www.influxdata.com/time-series-platform/) on [GCP](https://cloud.google.com/gcp/) using [Terraform](https://www.terraform.io/) and [Packer](https://www.packer.io/).
 
+## TICK Stack Architecture
+
+![TICK multi-cluster architecture](https://github.com/gruntwork-io/terraform-google-influx/blob/master/_docs/tick-multi-cluster-architecture.png?raw=true)
+
+## Features
+
+- Deploy the TICK stack (Telegraf, InfluxDB, Chronograf, Kapacitor) to gather and process time series data.
+- Supports both InfluxDB Enterprise and OSS
```

Comment:
I also had the following when I did nomad (https://github.com/hashicorp/terraform-aws-nomad), which I think also applies here:

- Supports colocated clusters and separate clusters
- Auto scaling and Auto healing